nservicebus/rabbitmq/routing-topology.md

```diff
@@ -54,7 +54,7 @@ Adjust the conventions for exchange name and routing key by using the overload:
 
 snippet:rabbitmq-config-usedirectroutingtopologywithcustomconventions
 
-WARNING: The direct routing topology is nondeterministic for message types with "non-system" interfaces in their inheritance hierarchy. For deterministic routing of these message types, the conventional routing topology must be used. A "non-system" interface is any which is not contained in a .NET Framework assembly (any assembly signed with the same public key as mscorlib), and is not one of the [marker interfaces](/nservicebus/messaging/messages-events-commands.md#defining-messages-marker-interfaces).
+WARNING: In some cases, the direct routing topology may not deliver message types with "non-system" interfaces in their inheritance hierarchy. A "non-system" interface is any interface which is not contained in a .NET Framework assembly (any assembly signed with the same public key as mscorlib), and is not one of the [marker interfaces](/nservicebus/messaging/messages-events-commands.md#defining-messages-marker-interfaces). When using the direct routing topology, messsage types must not inherit from "non-system" interfaces in order to guarantee delivery. To guarantee delivery of message types which inherit from non-system interfaces, the conventional routing topology must be used.
```

Comment:
The wording of the last sentence sounds a bit strange since, it is repeating the last three words of the previous sentence.

Perhaps "The conventional routing topology must be used to guarantee delivery of message types which inherit from non-system interfaces." instead?